actions/start-rest

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+source /etc/environment
+cd $ZOO_REST
```

Comment:
Let's check that $ZOO_REST is a dir before we go ant'ing stuff.  Please consider:

```
if [[ -d ${ZOO_REST} ]]; then
  cd ${ZOO_REST}
  ant stuff
fi
```